torch/utils/data/_utils/worker.py

```diff
@@ -201,7 +201,7 @@ def mix(x, y):
 
 def _worker_loop(dataset_kind, dataset, index_queue, data_queue, done_event,
                  auto_collation, collate_fn, drop_last, base_seed, init_fn, worker_id,
-                 num_workers, persistent_workers):
+                 num_workers, persistent_workers, shared_seed):
```

Comment:
nit: it seems like the argument `persistent_workers` isn't used? Do we still need it?

Reply:
I actually don't know. Probably not needed if not reference in this `worker_loop` function. Even if we want to remove it, I won't keep the change in this PR.